db.json

```diff
@@ -5680,17 +5909,27 @@
     "source": "apache",
     "extensions": ["mie"]
   },
+  "application/x-mif": {
+    "source": "mime-support",
+    "extensions": ["mif"]
+  },
   "application/x-mobipocket-ebook": {
     "source": "apache",
     "extensions": ["prc","mobi"]
   },
   "application/x-mpegurl": {
-    "compressible": false
+    "source": "mime-support",
+    "compressible": false,
+    "extensions": ["m3u8"]
```

Comment:
It was my understanding this extension is for the type "application/vnd.apple.mpegurl", right? Looking at the spec https://tools.ietf.org/html/rfc8216#section-4 the section says:

>    Each Playlist file MUST be identifiable either by the path component
   of its URI or by HTTP Content-Type.  In the first case, the path MUST
   end with either .m3u8 or .m3u.  In the second, the HTTP Content-Type
   MUST be "application/vnd.apple.mpegurl" or "audio/mpegurl".  Clients
   SHOULD refuse to parse Playlists that are not so identified.

Reply:
The mapping is not reversible. They both have the same extension?

Reply:
I'm not sure what your comment means, I'm sorry. Can you state it a different way?

Reply:
File with MIME type `application/x-mpegurl` and `application/vnd.apple.mpegurl` both can have ext m3u8.

Reply:
Yep, I get it. My question is can you site where the MIME type "application/x-mpegurl" comes from? I don't see it in the specification anywhere. Without a source, one could always argue that "foo/bar" MIME type is also m3u8 :)

Reply:
Not sure. To answer all of your questions, which are legit, will have to get more info on how this DB was pulled together, from the maintained. Don't expect you to merge with out that. Fact remains though this the Debian Linux `/etc/mime.types` file source, so in my mind that gives it as much legitimacy as the Apache or Nginx versions.

Reply:
Sure, but that doesn't mean they do not have outdated or invalid entries. When we pulled in NGINX and Apache back in the day, we did this same process and fixed a lot of bad data. That is what we'd need to do in this same case.

Reply:
Ah OK . That clears it up. Not sure if I have the time right now for all that!